### ISSUE TITLE
Alternative extensions for formats (jpg, jpeg, ...)

### ISSUE THREAD

**Issue by drozdik-m** [neilharvey/FileSignatures#60]
Unfortunately, to my current understanding, some file extensions such as jpeg/jpg or tiff/tif have various alternative extensions. I need to validate that the file extension matches the mime type, but it is rather difficult, since this library does not take into consideration this alternative extension stuff. 

Or, is it more complex than I think?

**Comment by drozdik-m** [comment]
There is even a conflict in README and code

![image](https://github.com/neilharvey/FileSignatures/assets/15088412/3fbac96f-a82e-483c-ba94-6bff3b2f6c31)

![image](https://github.com/neilharvey/FileSignatures/assets/15088412/2e88701b-4474-4bc6-a0ae-2c7ae56a15fb)

**Comment by neilharvey** [comment]
Yes, currently only a single extension is supported.  I don't like to make breaking changes but it is incorrect so perhaps I should change it.  

I would be tempted to keep the existing property so we don't break anyone's code but mark it as obsolete (for the duration of a major version) so we would have something like:

```cs
public IEnumerable<string> Extensions { get; }

[Obsolete("Use Extensions property instead")]
public string Extension { get; } => Extensions.First();
```

Would that solve your issue?

**Comment by drozdik-m** [comment]
Yes, that sounds phenomenal, thank you.

**Comment by tonyqus** [comment]
Has Extensions property been added to FileFormat class? I don't see it in the master branch

This issue looks to be duplicated with #64

**Comment by neilharvey** [comment]
@tonyqus Not yet, still on my list of things to do!

**Comment by Basssiiie** [comment]
Any update on this by any chance? We have some code that tries to check if the extension matches the signature using this library, and that fails now for JPEG, and it'd be great if we could handle that with this library as well instead of hardcoding an exception case.

Thanks for your time and hard work! 😊

**Comment by neilharvey** [comment]
@Basssiiie I've got something I'm working on, I need to go through all the formats and check what extensions are available.  I'll try to get it finished up and released :)
